### ISSUE TITLE
results on Android are very poor，but on PC they are very good

### ISSUE THREAD

**Issue by wzy123456wzy** [pytorch/android-demo-app#274]
The results on Android are very poor, but on PC they are very good. They only use the official conversion model, but the accuracy drops dramatically

**Comment by AnnaKholkina** [comment]
I have the same problem. Found a solution?

**Comment by xuyixun21** [comment]
+1

**Comment by nguyenviettuan96** [comment]
+1

**Comment by nguyenviettuan96** [comment]
maybe we should convert to tensorflow and run by importing 
org.tensorflow.contrib.android.TensorFlowInferenceInterface;
